ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -282,3 +282,35 @@ Check Pagination Is Correct On The Current Page
         ${expected_text}=    Set Variable   11 - ${total} of ${total}
     END
     Should Be Equal    ${expected_text}    ${actual_text}
+
+Check Resource Name Should Be Immutable
+    [Documentation]    Checks if the Resource Name is not editable
+    [Arguments]     ${project_title}
+    Click Action From Actions Menu    item_title=${project_title}    item_type=project   action=Edit
+    Wait Until Page Contains Element    ${RESOURCE_INPUT_XP}
+    Element Should Be Disabled    ${RESOURCE_INPUT_XP}
+    Click Button    Cancel
+
+Update And Verify Data Science Project Name
+    [Documentation]    Updates the existing DSG project name and verifies changes are done
+    [Arguments]     ${project_title}    ${new_title}
+    Click Action From Actions Menu    item_title=${project_title}    item_type=project   action=Edit
+    Wait Until Page Contains Element    ${DESCR_INPUT_XP}
+    Input Text    ${TITLE_INPUT_XP}    ${new_title}
+    Click Button    Update
+    Wait Until Generic Modal Disappears
+    ${is_open}=    Is Data Science Projects Page Open
+    IF    ${is_open} == ${FALSE}    Open Data Science Projects Home Page
+    Wait Until Page Contains    ${new_title}
+
+Update And Verify Data Science Project Description
+    [Documentation]    Updates the existing DSG project description and verifies changes are done
+    [Arguments]    ${project_title}    ${new_description}
+    Click Action From Actions Menu    item_title=${project_title}    item_type=project   action=Edit
+    Wait Until Page Contains Element    ${DESCR_INPUT_XP}
+    Input Text    ${DESCR_INPUT_XP}    ${new_description}
+    Click Button    Update
+    Wait Until Generic Modal Disappears
+    Open Data Science Project Details Page       project_title=${project_title}
+    Page Should Contain    ${new_description}
```

Comment:
could you move the verification step outside? maybe in another kw in the test suite file

Reply:
I updated the Update keyword and left the verification kw. Let me know if this fits.